app/views/users/_user_info.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="user-info-container">
+    <div class="bio">
+        <h2>Bio</h2>
+        <p><%= user.bio %></p>
+    </div>
+
+    <div class="display1 posts-expanded">
+        <% user.posts.each do |post, idx| %>
+            <div class="display1 post">
+                <h2><%= link_to "Post: #{post.title}", user_post_path(post.author, post) %></h2>
+                <p><%= post.text %></p>
+            </div>
+        <% end %>    
+    </div>
+    <div class="display2 posts-relaxed">
+        <% user.recent_three_posts.each do |post, idx| %>
+            <div class="post">
+                <h2><%= link_to "Post: #{post.title}", user_post_path(post.author, post) %></h2>
+                <p><%= post.text %></p>
+            </div>
+        <% end %>    
+    </div>
+    <button id="all">See all posts</button>
```

Comment:
- [ ] Great job so far @wacoo There is just one change I will suggest you implement and you'll be on your way to the next Milestone :wink: 
- [ ] Great job making sure the `See all Posts` button shows the hidden posts. But from the given wireframe and the project requirements, just as the previous reviewer mentioned you need to ensure that the `See all Posts` button opens the `Posts Index Page` and shows all the posts for a given user. So kindly work on that :+1:
- Your Implementation

[Screencast from 11-10-2023 12:17:44 PM.webm](https://github.com/wacoo/blog-front-ruby-rails/assets/102699958/d7b2d71f-3c47-4709-b7c6-a963505e04bf)

- The Project Requirements

![image](https://github.com/wacoo/blog-front-ruby-rails/assets/102699958/955428e7-3892-4eee-96be-c747861c2638)